umm-spec-lib/src/cmr/umm_spec/migration/version_migration.clj

```diff
@@ -156,11 +157,6 @@
       (update-in [:ContactPersons] #(mapv ci/contact-info-to-array %))
       (update-in [:ContactGroups] #(mapv ci/contact-info-to-array %))))
 
-(defmethod migrate-umm-version [:collection "1.6" "1.7"]
```

Comment:
I guess removing the whole migrate function isn't causing any problems. We might want to comment here why it was removed or just keep a dummy function in to return c. I think it would be confusing if someone down the road realized a whole migration was missing.